### ISSUE TITLE
Flatpack version 6.0.5.3 - opening a 1080p h264 file, nothing is displayed.

### ISSUE THREAD

**Issue by jason-a69** [ozmartian/vidcutter#392]
No thumbnails are generated

ffprobe returns this on the problematic file
```

  Stream #0:1[0x65]: Video: h264 (High) ([27][0][0][0] / 0x001B), yuv420p(tv, bt709, top first), 1920x1080 [SAR 1:1 DAR 16:9], 25 fps, 50 tbr, 90k tbn, 50 tbc
  Stream #0:2[0x66](eng): Audio: aac_latm (LC) ([17][0][0][0] / 0x0011), 48000 Hz, stereo, fltp
  Stream #0:3[0x6a](eng): Unknown: none ([17][0][0][0] / 0x0011) (visual impaired) (descriptions) (dependent)
  Stream #0:4[0x69](eng): Subtitle: dvb_subtitle ([6][0][0][0] / 0x0006)
  No Program
  Stream #0:0[0x12]: Data: epg
```

Log file contains this
```
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     self.loadMedia(filename)
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:   File "/app/lib/python3.10/site-packages/vidcutter/videocutter.py"
, line 989, in loadMedia
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     self.videoService.setMedia(self.currentMedia)
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:   File "/app/lib/python3.10/site-packages/vidcutter/libs/videoservi
ce.py", line 94, in setMedia
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     [stream for stream in self.media.streams if stream.codec_type =
= codec_type.lower()])
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:   File "/app/lib/python3.10/site-packages/vidcutter/libs/videoservi
ce.py", line 94, in <listcomp>
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     [stream for stream in self.media.streams if stream.codec_type =
= codec_type.lower()])
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:   File "/app/lib/python3.10/site-packages/vidcutter/libs/munch.py",
 line 104, in __getattr__
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     raise AttributeError(k)
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]: 2023-11-05 21:56 - root - CRITICAL - <class 'AttributeError'>: code
c_type
```
I do not mind downloading and installing from the source code to get this working...including ffmpeg if that is causing the problem.

Any feedback is welcome.

Thanks

Jason

**Comment by ozmartian** [comment]
Could you install mpv video player on your system and let me know if you can open and view the file with it. Trying to isolate the issue. And can you confirm that this issue only occurs with the one video file or if its happening with all videos.



Thanks






---- On Mon, 06 Nov 2023 01:05:30 +1100 jason-a69 ***@***.***> wrote ---





No thumbnails are generated

ffprobe returns this on the problematic file


  Stream #0:1[0x65]: Video: h264 (High) ([27][0][0][0] / 0x001B), yuv420p(tv, bt709, top first), 1920x1080 [SAR 1:1 DAR 16:9], 25 fps, 50 tbr, 90k tbn, 50 tbc
  Stream #0:2[0x66](eng): Audio: aac_latm (LC) ([17][0][0][0] / 0x0011), 48000 Hz, stereo, fltp
  Stream #0:3[0x6a](eng): Unknown: none ([17][0][0][0] / 0x0011) (visual impaired) (descriptions) (dependent)
  Stream #0:4[0x69](eng): Subtitle: dvb_subtitle ([6][0][0][0] / 0x0006)
  No Program
  Stream #0:0[0x12]: Data: epg


Log file contains this

Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     self.loadMedia(filename)
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:   File "/app/lib/python3.10/site-packages/vidcutter/videocutter.py"
, line 989, in loadMedia
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     self.videoService.setMedia(self.currentMedia)
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:   File "/app/lib/python3.10/site-packages/vidcutter/libs/videoservi
ce.py", line 94, in setMedia
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     [stream for stream in self.media.streams if stream.codec_type =
= codec_type.lower()])
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:   File "/app/lib/python3.10/site-packages/vidcutter/libs/videoservi
ce.py", line 94, in <listcomp>
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     [stream for stream in self.media.streams if stream.codec_type =
= codec_type.lower()])
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:   File "/app/lib/python3.10/site-packages/vidcutter/libs/munch.py",
 line 104, in __getattr__
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]:     raise AttributeError(k)
Nov  5 21:56:57 zolly com.ozmartians.VidCutter.desktop[40893]: 2023-11-05 21:56 - root - CRITICAL - <class 'AttributeError'>: code
c_type


I do not mind downloading and installing from the source code to get this working...including ffmpeg if that is causing the problem.

Any feedback is welcome.

Thanks

Jason

**Comment by jason-a69** [comment]
I just installed mpv and the video plays fine. 

It only fails on HD files, SD ones are fine. Would you like me to send you an example?

**Comment by ozmartian** [comment]
Yep, thanks. Just file share a small example file and post a link.



Thanks!






---- On Tue, 14 Nov 2023 16:46:07 +1100 jason-a69 ***@***.***> wrote ---





I just installed mpv and the video plays fine.

It only fails on HD files, SD ones are fine. Would you like me to send you an example?

**Comment by jason-a69** [comment]
Hi again,

Give this a go

https://jjactn.mooo.com/nextcloud/s/az2cZm35Rgxkdkb

**Comment by ozmartian** [comment]
Heya,



Link ain't working. ERR_CONNECTION_REFUSED









---- On Tue, 14 Nov 2023 17:13:04 +1100 jason-a69 ***@***.***> wrote ---





Hi again,

Give this a go

https://jjactn.mooo.com/nextcloud/s/az2cZm35Rgxkdkb

**Comment by jason-a69** [comment]
Sorry, can you try again, I have been bouncing that machine trying to fix ALSA problems.

It's up now, can you please grab the file, I won't restart it until you have.

**Comment by ozmartian** [comment]
Great. Working now and got the file. Will take a look when able soon.



Cheers






---- On Tue, 14 Nov 2023 19:20:04 +1100 jason-a69 ***@***.***> wrote ---





Sorry, can you try again, I have been bouncing that machine trying to fix ALSA problems.

It's up now, can you please grab the file, I won't restart it until you have.

**Comment by ozmartian** [comment]
Tested and all is working on my end. Only two thumbnails generated in the timeline due to the video length. This must be a Flatpak specific issue so will need to do more testing with flatpak later.



What Linux distro are you using by the way?



Thanks






---- On Tue, 14 Nov 2023 19:20:04 +1100 jason-a69 ***@***.***> wrote ---





Sorry, can you try again, I have been bouncing that machine trying to fix ALSA problems.

It's up now, can you please grab the file, I won't restart it until you have.

**Comment by ozmartian** [comment]
One more thing. The flatpak was updated a day or so ago so make sure to try upgrading it as the latest update includes a newer FFMpeg 6.1 plus many other updates.



Thanks






---- On Tue, 14 Nov 2023 19:20:04 +1100 jason-a69 ***@***.***> wrote ---





Sorry, can you try again, I have been bouncing that machine trying to fix ALSA problems.

It's up now, can you please grab the file, I won't restart it until you have.

**Comment by jason-a69** [comment]
Just download flatpak v6.5.0.3 on Pop OS (which looks to be about a month old) from the Pop OS shop and that did not work.

I tried
```
flatpak install flathub com.ozmartians.VidCutter
flatpak run com.ozmartians.VidCutter
```

And that installed v6.5.0.1

I am still getting errors with both versions but I am guessing that I do not have the latest version?
